src/tools/dev/scripts/bv_support/bv_llvm.sh

```diff
@@ -92,10 +95,25 @@ function bv_llvm_ensure
 {
     if [[ "$DO_DBIO_ONLY" != "yes" ]]; then
         if [[ "$DO_LLVM" == "yes" ]] ; then
-            ensure_built_or_ready "llvm"   $BV_LLVM_VERSION   $BV_LLVM_BUILD_DIR   $BV_LLVM_FILE $BV_LLVM_URL
+            check_installed_or_have_src "llvm" $LLVM_VERSION $LLVM_BUILD_DIR $LLVM_FILE $LLVM_URL
             if [[ $? != 0 ]] ; then
                 return 1
             fi
+
```

Comment:
Hmm...this looks like a cut-n-paste of some code from `helper_funcs` here. Why?

Reply:
A new line was added???